### PR TITLE
added news on Zephyr port

### PR DESCRIPTION
**Pull request by azoitl**
@dok-net  as promissed here is my proposed news post. Thx in advance for cross checking.

### CODE REVIEW

File: news/en_news_20240320.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+This integration enables industrial automation developers to leverage the power of Zephyr in their projects within the Eclipse 4diac ecosystem. 
+</p>
+
+<p>
+With the addition of Zephyr support, Eclipse 4diac users gain access to a robust and scalable real-time operating system that is perfectly suited for applications in the industrial automation domain. 
+Zephyr's lightweight and efficient architecture allows it to run on a wide range of microcontrollers and small footprint devices, making it an ideal choice for resource-constrained environments.
+</p>
+<p>
+To get started with Eclipse 4diac's Zephyr support, get the latest 4diac FORTE development branch from 
+<a href ="https://github.com/eclipse-4diac/4diac-forte/tree/develop">4diac FORTE's Github repositorie</a> and the documentation provided in
```
Comment (kaartee) [inline news/en_news_20240320.php:16]: Typo: repositorie -> repository

**Comment by kaartee** [comment]
Maybe mention where to get a compatible 4diac IDE as well? Nightly builds are what I am used to, now.

File: news/en_news_20240320.php (hunk 1 above)
Comment (azoitl) [reply news/en_news_20240320.php]: thx for noting. fixed

**Comment by azoitl** [comment]
> Maybe directly mention where to get a compatible 4diac IDE as well? Nightly builds are what I am used to, now.

Good point. Added a paragraph on that. For starters I think the Milestone versions are a bit better, therefore I used the milestone link.

File: news/en_news_20240320.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+</p>
+<p>
+To get started with Eclipse 4diac's Zephyr support, get the latest 4diac FORTE development branch from 
+<a href ="https://github.com/eclipse-4diac/4diac-forte/tree/develop">4diac FORTE's Github repository</a> and the documentation provided in 
+<a href="https://github.com/eclipse-4diac/4diac-forte/blob/develop/src/arch/zephyr/README.md">the Readme.MD</a> for 4diac FORTE’s Zephyr directory.
+</p>
+
+<p>
+For using the latest 4diac FORTE develop version you also need a newer 4diac IDE. 
+We recommand using our <a href="https://download.eclipse.org/4diac/updates/milestone/">pre-built milestone version</a>, which give you a sneak preview on the amazing upcoming 4diac IDE features.
```
[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: thx, this happens when I rush. Fixed

[the remark this answers is not in the document]
Comment (dok-net) [reply news/en_news_20240320.php]: Please wait for my final go ahead until tomorrow afternoon, I am in the process of checking with corporate management.

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: Sure. That's exactly the reason I wanted to have you in the loop here.

[the remark this answers is not in the document]
Comment (kaartee) [reply news/en_news_20240320.php]: Here's the feedback I get:

I was asked to express that "a portion of the Zephyr port was developed as part of my employment with KT Elektronik."
This cannot be construed as a commitment by KT Elektronik to use or support 4diac in the future.

Please write something nice about it. I'll personally check the final pre-merge commit on request by you.

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: Sure I'll draft something and ping you. May I add a link to KT Elektronik?

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: @dok-net what do you think of my attempt?

[the remark this answers is not in the document]
Comment (kaartee) [reply news/en_news_20240320.php]: I'm not happy with "sponsored". Bing corroborates that a phrase along the lines of "contribution was made as part of employment with" is common wording. If you could adapt the news release to express that situation clearly, I shall clear it ASAP.

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: @dok-net is it now better.

[the remark this answers is not in the document]
Comment (kaartee) [reply news/en_news_20240320.php]: Just fix the typo "recommand" (recommend) in line 28.

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: :man_facepalming:  fixed

[the remark this answers is not in the document]
Comment (kaartee) [reply news/en_news_20240320.php]: For the initial news posting, I guess this is final version, so from my point of view, please go ahead.

[the remark this answers is not in the document]
Comment (azoitl) [reply news/en_news_20240320.php]: Thx.

**Pull request merged**